src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py

```diff
@@ -220,11 +222,13 @@ def __discoverCredentialsToUse(self):
         else:
             self.__useCertificates = gConfig.useServerCertificate()
             self.kwargs[self.KW_USE_CERTIFICATES] = self.__useCertificates
-        if self.KW_SKIP_CA_CHECK not in self.kwargs:
-            if self.__useCertificates:
-                self.kwargs[self.KW_SKIP_CA_CHECK] = False
-            else:
-                self.kwargs[self.KW_SKIP_CA_CHECK] = skipCACheck()
```

Comment:
Was it important that this was filling a value into the `self.kwargs` ?

Reply:
Nope. I think it's just historical legacy from when the class was copied from DISET. Ideally it needs to be simplified but that can wait a little longer.